base/BUILD

```diff
@@ -41,10 +43,30 @@ docker_build(
     ],
 )
 
-load("@runtimes_common//structure_tests:tests.bzl", "structure_test")
-
 structure_test(
     name = "base_test",
     config = "testdata/base.yaml",
     image = ":base",
 )
+
+# Test that the /etc/passwd entry is set and that libc works (for Go)
+
+go_binary(
+    name = "user",
+    srcs = ["testdata/user.go"],
+    visibility = ["//visibility:private"],
+)
+
+docker_build(
```

Comment:
I started with it in examples, but I changed my mind. The idea is that this is supposed to be a test of the base image's glibc installation, so I thought it would make more sense here. But it is also sort of an example...

WDYT makes more sense?